tests/vmi_networking_test.go

```diff
@@ -108,11 +108,14 @@ var _ = Describe("Networking", func() {
 			outboundVMI := tests.NewRandomVMIWithEphemeralDiskAndUserdata(tests.RegistryDiskFor(tests.RegistryDiskCirros), "#!/bin/bash\necho 'hello'\n")
 
 			if withPodNetwork {
+				// add explicit pod network interface
 				v1.SetDefaults_NetworkInterface(inboundVMI)
 				v1.SetDefaults_NetworkInterface(outboundVMI)
-				for _, networkVMI := range []*v1.VirtualMachineInstance{inboundVMI, outboundVMI} {
+				for i, networkVMI := range []*v1.VirtualMachineInstance{inboundVMI, outboundVMI} {
 					Expect(networkVMI.Spec.Domain.Devices.Interfaces).ToNot(BeZero())
 					Expect(networkVMI.Spec.Networks).ToNot(BeZero())
+					// also set custom MAC address to the interface
+					networkVMI.Spec.Domain.Devices.Interfaces[0].MacAddress = fmt.Sprintf("de:ad:00:0%d:be:af", i)
```

Comment:
with this addition, do we have any functional tests that verify connectivity that don't explicitly set the mac now?

Reply:
No, though we have checks for "implicit" case (no interfaces in spec, so default one is used); I will reinstate it, but consider that the test matrix starts to blow up if we add all connectivity tests to each variant of configuration. Test run time is not ideal already, I was thinking maybe doing these test matrix multiplications is problematic.